model/src/w3fldsmd.F90

```diff
@@ -203,6 +203,7 @@ SUBROUTINE W3FLDO ( INXOUT, IDFLD, NDS, NDST, NDSE, NX, NY,     &
 #ifdef W3_DEBUGFLS
       USE W3ODATMD, only : IAPROC
 #endif
+      use constants, only: file_endian
```

Comment:
I just removed this line + two other additions in model/src/w3fldsmd.F90 and tested regtests with both little and big endians. It seems the fix from the finished jobs. I'll let you know once all tests are done, so you can revert them.